.rubocop_todo.yml

```diff
@@ -504,7 +504,7 @@ Rails/InverseOf:
     - 'app/models/spree/variant.rb'
     - 'app/models/subscription_line_item.rb'
 
-# Offense count: 38
+# Offense count: 35
```

Comment:
Updates like this one are not necessary to include in your PR and they can cause conflicts. Maybe reduce your commits the actual change of your PR to avoid conflicts.